src/tools/debugger/nodes/BreakpointPromiseRcvNode.java

```diff
@@ -0,0 +1,39 @@
+package tools.debugger.nodes;
```

Comment:
Please refactor these classes to avoid code duplication. The only thing that seems to be different is how the BreakpointEnabling<T> object is retrieved. This can be factored out.